libbinlogevents/include/binlog_event.h

```diff
@@ -129,10 +130,27 @@
    1 + 255 /* host_len, host */ + 1U + 1 /* type, explicit_def..ts*/ + 1U +   \
    8 /* type, xid of DDL */ + 1U +                                            \
    2 /* type, default_collation_for_utf8mb4_number */ + 1U +                  \
-   1 /* type, sql_require_primary_key */ + 1U +                                     \
+   1 /* type, sql_require_primary_key */ + 1U +                               \
    1 /* type, default_table_encryption */ + 1U +                              \
-   1 /* type, binlog_ddl_skip_rewrite */)
-
+   1 /* type, binlog_ddl_skip_rewrite */ + 1U +                               \
+   1 /* type, wsrep_applier_skip_readonly_checks */)
```

Comment:
Is this flag transparent for PXB and other binlog-related tools? In other words: If we add this flag, does it impact any other external tool?

Reply:
No. I don think this will impact any other tool unless any of the dependent tool has a conflicting enum value in `Query_event_status_vars` enum.

Reply:
What if upstream adds another flag to this list in a later version, won't that cause backwards compatibility issues for PXC? (I see that we already added a flag to this list earlier)

Reply:
This will be a problem only when we replicate from a new upstream server to old pxc server. But this is not supported generally.

If upstream adds a new flag, then we must ensure that our changes are added in the end during our merges. This way there won't be any issues between upstream and PXC.

Reply:
I was thinking about the following situation:
* new flag added
* we release new PXC version
* user starts to upgrade, upgrades nodes one-by-one
* for a short period, we can have a situation where different nodes in the cluster run different versions, and the flag values have different meanings

Reply:
It was decided on slack to leave the old flags (added by us) as they are and to add the new flag after a reasonable gap. A value of 128 was chosen for the new flag to ensure we have enoug gaps for upstream flags.